### PR TITLE
feat(docker): ability to project inside container

### PR DESCRIPTION
**Pull request by shootermv**
Ability to run project inside docker container #385 

## What:

Issue number: N/A

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+RUN mkdir /usr/src/app/cypress/plugins
+WORKDIR /usr/src/app
+
+# add `/usr/src/app/node_modules/.bin` to $PATH
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
+
+# install and cache app dependencies
+COPY package.json /usr/src/app/package.json
+RUN npm install --silent
+RUN npm install @angular/cli@~6.1.3 -g --silent
```
Comment (tomastrajan) [inline Dockerfile]: Just to understand, why do we need also global `@angular/cli` ?

Comment (tomastrajan) [reply Dockerfile]: Also can you please provide a bit of more comment on how this works?

**Comment by tomastrajan** [comment]
@shootermv can u please provide bit more details on this or even better, update `README.md` so that people know how to run it? (maybe with some links to base tutorial about docker if its too much to write inline? )

**Comment by shootermv** [comment]
Hi Tomas. Did you notice the current readme short description i added at
readme aleady?
Thanks

**Comment by tomastrajan** [comment]
@shootermv haha no, my bad! :)

**Pull request merged**